ci-operator/step-registry/upi/install/gcp/by-infra-manager/upi-install-gcp-by-infra-manager-commands.sh

```
@@ -0,0 +1,729 @@
+  local -r deployment_name="$1"; shift
+  local -r cmd_ret="$1"; shift
+  local CMD
+
+  set +e
+  echo "$(date -u --rfc-3339=seconds) - Debugging for infrastructure manager deployment '${deployment_name}' (${cmd_ret})"
+  CMD="gcloud infra-manager deployments describe ${deployment_name} --location=${REGION}"
+  run_command "${CMD}"
+  set -e
+
+  echo "$(date -u --rfc-3339=seconds) - Should exit for command code '${cmd_ret}'"
+  exit ${cmd_ret}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)